frontend/packages/integration-tests-cypress/tests/app/demo-dynamic-plugin.spec.ts

```diff
@@ -96,99 +96,104 @@ const k8sAPINavTest = (apiID: string) => {
   cy.get('test-k8api-error').should('not.exist');
   cy.get(`test-k8s-${apiID}`).should('not.be.empty');
 };
+if (Cypress.env('OPENSHIFT_BUILD_NAME') !== 'console-operator') {
```

Comment:
Better to test for `PLUGIN_PULL_SPEC`? Then we don't have to hard-code build names.

Reply:
@spadgett the plugin pull spec doesn't exist when running e2e tests locally. So it would skip local testing. Do you want me to keep it as is or test for a ci env AND absence of pull spec?

Reply:
Yes, I would check for

```suggestion
if (Cypress.env('OPENSHIFT_CI') && Cypress.env('PLUGIN_PULL_SPEC')) {
```

Reply:
@spadgett you got it!